modules/uhttp/http_options.go

```diff
@@ -20,17 +20,17 @@
 
 package uhttp
 
-// ModuleOption is a function that configures module creation.
-type ModuleOption func(*moduleOptions) error
+// ModuleOptionFn is a function that configures module creation.
+type ModuleOptionFn func(*moduleOption) error
```

Comment:
Lets separate this change from the intended changes in the PR title?

There are some other options we can use and implement. I do like Zap interface with private functions.